en/docs/tryout-flows/accelerator-with-is-and-apim/try-out-payments-flow.md

```
@@ -154,6 +154,23 @@ The response contains a Consent ID. A sample response looks as follows:
 }
 ```
 
+#### With UK JWS Header Processing Policy
+
+If you have enabled the [UK JWS Header Processing Policy](../../learn/uk-jws-header-processing-policy.md), the payload of this request must be digitally signed using your TPP Private Key. You must generate a Detached JWS (Header..Signature) and pass it in the `x-jws-signature` HTTP header. The payload used to generate the signature **must exactly match** the JSON body of your request (no extra spaces or newlines).
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)